classes/controller/menu/class-enform-post-controller.php

```diff
@@ -177,6 +177,26 @@ public function handle_form_shortcode( $atts ) {
 					'questions' => $questions,
 					'optins'    => $optins,
 				];
+
+				$supporter_fields_map = [
+					'title'       => 'Title',
+					'first'       => 'First name',
+					'last'        => 'Last name',
+					'address1'    => 'Address 1',
+					'address2'    => 'Address 2',
+					'city'        => 'City',
+					'country'     => 'Country',
+					'post'        => 'Postcode',
+					'email'       => 'Email',
+					'region'      => 'County',
+					'phoneNumber' => 'Phone Number',
+					'birth'       => 'birthday'
```

Comment:
phpcs error: every line must end in a comma
```suggestion
					'birth'       => 'birthday',
```